frontend/src/views/ImageDetailsPage/ImageDetailsPage.js

```diff
@@ -11,40 +11,54 @@ import ImageDetailsArea from 'views/ImageDetailsPage/components/ImageDetailsArea
 
 const ImageDetailsPage = () => {
   const { id } = useParams();
-
-  const [isLoading, setIsLoading] = useState(true);
+  const [imageDetails, setImageDetails] = useState(null);
+  const [isImageLoading, setIsImageLoading] = useState(true);
+  const [isDescriptionLoading, setIsDescriptionLoading] = useState(true);
```

Comment:
I think `false` as a default would be a better choice - now this "images are..." message is also shown when I go for nonexisting image id, like `/ImageDetails/kk`.